src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -48,6 +48,24 @@ module SoilBiogeochemDecompCascadeBGCMod
   integer, private :: i_cel_lit  ! index of cellulose litter pool
   integer, private :: i_lig_lit  ! index of lignin litter pool
 
+  real(r8), private :: cwd_fcel
+  real(r8), private :: cwd_flig
+  real(r8), private, allocatable :: f_s1s2(:,:)
+  real(r8), private, allocatable :: f_s1s3(:,:)
+  real(r8), private :: f_s2s1
+  real(r8), private :: f_s2s3
+
+  integer, private :: i_l1s1
+  integer, private :: i_l2s1
+  integer, private :: i_l3s2
+  integer, private :: i_s1s2
+  integer, private :: i_s1s3
+  integer, private :: i_s2s1
+  integer, private :: i_s2s3
+  integer, private :: i_s3s1
+  integer, private :: i_cwdl2
+  integer, private :: i_cwdl3
```

Comment:
I made these variables available to this entire module to accommodate my changes to `pathfrac_decomp_cascade`. Similarly in `SoilBiogeochemDecompCascadeCNMod.F90`.

Reply:
are the parallel changes needed in CNMod?

Reply:
I made the changes in CNMod to avoid errors when building the code. Otherwise no.